### ISSUE TITLE
Add dark mode support to the `?v=product` version of the Firefox PN and TOU.

### ISSUE THREAD

**Issue by stephaniehobson** [mozilla/bedrock#16302]
The in-product display of the Firefox Privacy Notice and Terms of Use should be enhanced to support dark mode.

https://bugzilla.mozilla.org/show_bug.cgi?id=1962581
